tests/ui_tools/test_boxes.py

```diff
@@ -17,6 +17,24 @@ def write_box(self, mocker):
         write_box = WriteBox(self.view)
         return write_box
 
+    @pytest.fixture()
+    def write_box_users(self, mocker, users_fixture, user_groups_fixture,
+                        write_box):
+        write_box.view.users = users_fixture
+        write_box.model.user_group_names = [
+            groups['name'] for groups in user_groups_fixture]
+        return write_box
+
+    @pytest.fixture()
+    def write_box_streams(self, mocker, streams_fixture, write_box):
```

Comment:
These new local fixtures seem OK, but it may be worth investigating if it would be more straightforward to extract common fixtures related to the groups and pinned streams.

Reply:
You mean combine the fixtures for streams and groups in the writebox fixture?

Reply:
Well, I recall pinning tests doing something similar to what you have here.